os_migrate/plugins/module_utils/serialization.py

```diff
@@ -116,3 +121,65 @@ def _recursive_trim(obj):
             return obj
 
     return _recursive_trim(resource)
+
+
+class Resource:
```

Comment:
As discussed on IRC, i think this approach would give us trouble on import, i will propose an idea PR as well to put my IRC words into some code to read :)